README.md

```diff
@@ -146,6 +128,13 @@ The following conventions are nothing more than that and are currently not enfor
 
 ## Changelog
 
+* 1.2.2-4
+
+  - `buildinfo.txt` did not correctly reflect the version 1.2.2
+  - Turn off debugging options when sourcing environment files
```

Comment:
das war doch bereits in <1.2.2-4 ausgestellt?

Reply:
No. This is also new compared to 1.2.2-3. `set -ue` had already be turned off before, but they are not really "debugging" options, because they constitute rather clean-coding rules, similar to Perl's "use strict" that should always be turned on. They had been turned off because external scripts like `/etc/profile` or Conda's environment scripts do not adhere to clean coding rules in Bash (for the sake of simplicity). Here only `set +vx` are added to reduce the otherwise extreme debugging output.